compliance/collection/inventory/convert.go

```diff
@@ -0,0 +1,110 @@
+package inventory
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/uuid"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+)
+
+// NodeInventoryResponseToNodeInventory converts a NodeInventory response to a native storage.NodeInventory version
+func NodeInventoryResponseToNodeInventory(response *scannerV1.GetNodeInventoryResponse) *storage.NodeInventory {
```

Comment:
How about the name: `ToNodeInventory(resp)`?

Reply:
Done!